app/controllers/api/v1/saved_scenarios_controller.rb

```diff
@@ -18,7 +18,14 @@ def index
       end
 
       def show
-        render json: current_user.saved_scenarios.find(params.require(:id))
+        saved_scenario = current_user
```

Comment:
What happens if there is no current user? E.g. somebody wants to load a featured scenario when not logged in?